### DISCUSSION TITLE
게임 후기

### DISCUSSION THREAD

**Discussion by wooyn730** [GG-Studio-990001/Grayed-Game#389] [General]
게임 후기들을 나눠봐요~~
아무 게임 아무 후기 아무나 써보기
스포 주의

**Comment by wooyn730** [comment]
**제목**: 스테퍼 케이스 (데모)
**플레이타임**: 약 50분
하게된 계기: 챕터2가 비주얼 노벨 + 정안님 추천

**요약**:
다른 추리 게임과 차별성은 초능력(스킬) 세계관이라는 점
주인공 빼고 동료, 용의자, 사물 등 초능력자가 매우 많음
주인공은 머글인데 추리력;;만으로 살아남아야함

**후기**:
비주얼 노벨 정말 간만에 하는 듯 정말 글이 많다
아트, 특히 캐디가 귀여운데 뭔가 인디 느낌이 나는 디자인..
증거에서 모순을 찾아내는 점에서 역전재판과 유사성을 느낌
은근 괜찮은 것 같음

아쉬운 점은 
1) 효과음.. 언더테일이나 역전재판의 경우 성우가 없더라도 대사 효과음(타이핑 사운드)에는 조금씩 변화를 줌
(전부는 아니어도) 남자면 좀 낮은 효과음, 여자면 좀 높은 효과음
(언더테일은 브금이랑 폰트에도 변화를 주기도 함)
비주얼 노벨 특성 상 글씨가 많은데 효과음도 적어서 아쉬웠음 조금 지루
2) 중간중간 logical!이라는 화면이 뜨는데 씬 전환 화면인가? 느려서 조금 더 루즈해지는 느낌
3) 인디 느낌이 왜 나나 생각해봤는데 UI 때문인가? 설정창이라던가 대화창이라던가
![image](https://github.com/GG-Studio-990001/Grayed-Game/assets/80298207/c238cce9-dfae-40d9-8cb4-32b9da2682d8)

기억 남는 점은
1) 연출이 나올 때는 위 아래 레터박스를 생성해서 옵션을 아예 가려둔 것 (세이브, 로드, 타이틀로 이동 등)
2) 설정에서 창 크기, 해상도 등을 다양하게 선택 가능하게 한 것

암튼 꽤 재밌는 듯
정안님 블로그 가서 후기 보고왔는데 뒷부분 쩐다고 하니 조금 사고싶어짐

**Reply by royalbluesm** [reply]
이거 흑막이 누구냐면...

**Reply by wooyn730** [reply]
안돼!!!!!!!!!!!!!

**Reply by fkdl0048** [reply]
스테퍼 케이스는 아직도 제 [게임리뷰](https://github.com/fkdl0048/GameReview)중 평점이 1등인 게임입니다. 제대로 즐기고 싶다면 4~5챕터를 구매 후 플레이 하시길!

**Comment by wooyn730** [comment]
**제목**: Kinito Pet
**플레이타임**: 1시간 반
**하게된 계기**: 이우님 추천 + 메타픽션 게임 해보고싶었음

**요약 (약 스포)**:
![image](https://github.com/GG-Studio-990001/Grayed-Game/assets/80298207/08fba467-efc9-4b21-bc55-caa22669ceba)
살짝 이런.. 바탕화면에 돌아다니는 캐릭터인데
나랑 평생 함께하고 싶어함 얀데레임(?)
"바탕화면에 존재"한다는 특성으로 내 컴퓨터 프로그램(카메라 등), cmd, 스팀 친구 등을 다 뒤져서 게임 요소로 활용함

**후기**:
메타픽션이라는 소재인데 주인공이 바탕화면에 있기 때문에 내 컴퓨터를 조작할 수 있는 명분이 된 달까?
다른 메타픽션보다 더 컴퓨터에 결합(?)되기 좋은 세계관
내 컴퓨터를 해킹?하는 느낌을 통해 확실히 제4의 벽을 넘는 느낌이 듦

_cmd를 켜서 자기 마음대로 명령어를 입력하는 부분은 진짜 내 컴퓨터를 조작하는 것 같아서 조금 소름돋았음_

(스포 시작)
나한테 질문을 하면서 내가 좋아하는 게 무엇인지 알아내고 내 그림을 수집한 뒤에
이를 후반부에 활용함
확실히 상호작용한다는 느낌 + 얀데레 느낌이 강하게 들어서 인상적이었음
(스포 끝)

은근 3D 요소가 많은데 컴퓨터 그래픽스 과제로 만든 것마냥 허접한 3D 시스템들이
이 게임의 전체적인 느낌에 잘 어울렸음

한 번 깨니까 세이브 포인트들을 여러개 보여줘서 중간부터 다시해서 다른 엔딩을 볼 수 있는 편의성을 제공함 => 좋음

아쉬운 점
눈뽕 요소가 꽤 많음 눈뽕 요소는 무섭지도 않고 눈만 나빠질 것 같아서 별로임
별로 안 무서움 (불 끄고 했어야 했나)
스토리가 강하진 않음 떡밥만 많고 걍 얀데레+메타픽션이 주 소재 (이런 점에선 두근두근 문예부도 조금 생각남)

그래도 플탐 짧고 가격 적당해서 메타픽션 호러게임 가볍게 해보고 싶으면 할 만 한 듯
애초에 컴퓨터 바탕화면에서 게임이 시작되고, 컴퓨터 밖의 내가 주인공이라는 점에서 메타픽션을 제대로 담았고 잘 살렸다고 생각함

**Reply by fkdl0048** [reply]
이 게임도 조금 알아봤는데, 메타적인 요소가 플레이어+컴퓨터라면 저희는 플레이어 + 게임이라는 요소로 분리가 가능한 것 같네요

**Comment by Sumindd** [comment]
### **Chants of Sennaar**

너무 재밌게 했고, 제 기획 추구미라서.. 제가 추구하는 기획 방향성을 보여줄 수 있는 기회인 것 같아 [블로그에 작성한 후기](https://blog.naver.com/min_d109/223397672526)를 긁어왔습니다!


플레이 인증
![image](https://github.com/GG-Studio-990001/Grayed-Game/assets/84775854/c2b8c6d4-8ae4-42ac-ab41-c4f2be22c3dc)
3번 나눠서 13.3시간 플레이하고 엔딩을 봤다.
너무너무 재밌게 플레이 했다.


​감상 요약
![image](https://github.com/GG-Studio-990001/Grayed-Game/assets/84775854/6228df03-0b14-48cf-a50e-43ab15c6353a)



​​

​스포 없는 선에서 캡쳐한 이미지
![image](https://github.com/GG-Studio-990001/Grayed-Game/assets/84775854/c5a17da2-42dd-4dfb-bc6e-4dc737c41d8e)
![image](https://github.com/GG-Studio-990001/Grayed-Game/assets/84775854/2a3d2a1f-8366-431f-9942-5b6ce52fee17)
![image](https://github.com/GG-Studio-990001/Grayed-Game/assets/84775854/9ff58100-f3b0-4b60-8c1d-3541ff5b56a7)
![image](https://github.com/GG-Studio-990001/Grayed-Game/assets/84775854/384a3707-021c-4cec-a627-0bb0cbd91997)
나의 단어 추리가 황당해서 캡쳐함 ㅋㅋㅋ









![image](https://github.com/GG-Studio-990001/Grayed-Game/assets/84775854/05b39641-ec21-4346-ae32-15b483544bdc)
![image](https://github.com/GG-Studio-990001/Grayed-Game/assets/84775854/32510b6c-3462-4e92-a348-4ffa11ba3a2b)
![image](https://github.com/GG-Studio-990001/Grayed-Game/assets/84775854/0d1a6bcb-2797-481a-abf6-0132df58a6d7)
![image](https://github.com/GG-Studio-990001/Grayed-Game/assets/84775854/0ed8990b-da42-4547-9ece-6e687f5c5150)
![image](https://github.com/GG-Studio-990001/Grayed-Game/assets/84775854/fd3efd50-b452-4e91-9343-2177ec2d8276)
색감이 미쳤다..



​

​

> 그래픽

확고한 컨셉, 층별로 색감이 달라서 다음 층은 어떨지 기대하게 된다. 색감이 너무 조화롭고 예쁘다.
UI가 너무 깔끔하고 최소한으로 줄여서 화면을 배경 그래픽으로 가득 채운 점도 좋았다. 
로우폴리 3D라서 단순하지만 규모감과 밀도가 있어서 단순해보이지 않고 꽉꽉 찬 느낌이다. 맵 탐방만 해도 즐겁다. 맵의 상황에 따라 특이한 카메라 구도가 많은데 연출적으로도 너무 좋았다.. 멀리서 봐야할 때는 줌아웃해서 건축물의 규모감을 느끼게 해주고, 특수한 각도의 구조에서는 그 구조의 매력을 극대화 시키는 구도로 보여줘서 좋았다. 3D 멀미 없이 깔끔하게 카메라 잡아줘서 불편하지 않게 플레이 했다.


독특한 카메라 구도1
![image](https://github.com/GG-Studio-990001/Grayed-Game/assets/84775854/d9886cd9-ee06-41bc-8718-5af406187cef)

![image](https://github.com/GG-Studio-990001/Grayed-Game/assets/84775854/8feda5d8-5f10-4dfb-9b92-dcad8583394a)
2

![image](https://github.com/GG-Studio-990001/Grayed-Game/assets/84775854/d034fbc6-3f72-487b-ace6-e72da89cca7d)
3

​

> 기획

종합적으로 시스템이 내 추구미다..
뼈대가 되는 시스템인 언어 추리/기록 시스템이 너무 군더더기 없이 깔끔하고 아이디어도 좋다.
맵 기획(구조)도 정말 좋다.. 적당한 수준으로 돌아오고, 구조가 크게 복잡하지도 않다. 나는 엄청난 길치인데도 생각보다 덜 헤맸다. (지하 수로에서는 좀 힘들었지만 딱 1가지 맵이고 지하라는 특수성이 있으니 한 번쯤은 괜찮다고 생각한다.)

​딱 한 가지 시스템적으로 개선했으면 싶었던 부분은..
![image](https://github.com/GG-Studio-990001/Grayed-Game/assets/84775854/d94bbc02-df49-4916-923f-bf0dfcbb79cc)
완성하지 않아도 책에 끼웠을 때 내 추리 메모가 보였으면 좋겠다. 그런데 이건 내가 바보라서 내가 끼우고 까먹거나.. 때려 맞춰서 끼우다가 까먹은 경우에 불편했던 부분이라 때려 맞추지 말라고 의도적으로 보여주지 않은 것 같기도 하다.

​나는 스크립트 없이 이야기를 담고 있는 스토리 게임을 좋아해서 각 층마다 소소하게 시스템을 통해 습득하게 되는 문명의 스토리도 흥미로웠다.  그리고 퍼즐의 다양성이나 난이도도 적당하고 복잡하지 않아서 레벨 디자인도 적당하다고 느꼈다. 
층 사이에서 번역을 하는 기믹을 풀면서 크게 느낀 점은, 각 언어마다 실제로 존재하는 그리고 존재하지 않지만(그냥 내가 모르는 것일지도..) 납득 가능한 방식의 언어 구조(특히 어순이)가 재미있게 다가왔다.  

​

> 총평

​이 게임을 한 이후로 이 게임처럼 새로운 시각으로 세상을 바라보고, 그걸 시스템으로 녹여낸 게임을 언젠가 꼭 만들어보고 싶다는 생각을 했다.. 나에게 정말 많은 긍정적인 영향을 준 게임이다. 강력하게 추천합니다!

​

**Reply by fkdl0048** [reply]
카메라 워킹이 정말 신기하네요,  색감도 그렇고 스토리적인 부분이 궁금해지는 게임인 것 같습니다.

**Reply by wooyn730** [reply]
기획 추구미라니 너무 궁금하네요,,;;;
저는 게임 대사 추구미(?)가 언더테일인데 조만간 후기 써야겠다...
